modules/core/package.json

```diff
@@ -16,10 +16,10 @@
   ],
   "scripts": {},
   "dependencies": {
-    "@deck.gl/core": "^8.1.5",
-    "@deck.gl/layers": "^8.1.5",
-    "@deck.gl/mesh-layers": "^8.1.5",
-    "@deck.gl/react": "^8.1.5",
+    "@deck.gl/core": "8.1.8",
```

Comment:
We should not be using fixed versions.  Why is this necessary.  Is there a patch version that is required?  Otherwise ^ should cover this already.

Reply:
I partially agree with you, unfortunately dependencies among the whole viz.gl packages are complex and having fuzzy versions leads to tons of conflicts. Some of them can be solved with resolutions, but a lot of them cannot.

Reply:
we've made it this far? Do we have a concise statement of the problem?  This does not seem to be a streetscape.gl issue, but an application issue. So the solution should be placed there, not in streetscape.gl